docs/guides/medical.md

```diff
@@ -0,0 +1,181 @@
+
+Przyjmuje się, że w razie uszczuplonych zapasów krew podaje się tylko nieprzytomnym lub osobom ze znaczną utratą krwi, a tym samym granicznym ciśnieniem krwi w aspekcie przytomności. Każdy członek drużyny powinien w miarę możliwość posiadać przy sobie 0,5 - 1 l krwi, co nie tylko odciąży medyka, ale również pozwoli medykowi korzystać z zapasów rannego. 
+
+W kwestii przetaczania krwi można sugerować się poniższą rozpiską, ale nie jest to sztywna zasada:
+- Stracił trochę krwi - nie podajemy krwi, ponieważ w zdecydowanej większości przypadków ciśnienie krwi oraz tętno są w granicach normy 
+- Stracił dużo krwi - podajemy 0,5 l
+- Stracił bardzo dużo krwi – podajemy 1 l
+- Stracił krytyczną ilość krwi – podajemy 1,5 l
+
+## Ciśnienie krwi i tętno
+Ciśnienie oraz tętno są właściwie najważniejszymi czynnikami w Medicalu. To one są wskaźnikiem w jakim stanie jest organizm żołnierza. Rolą medyka jest rozpoznanie za pomocą pomiaru ciśnienia oraz tętna jakiej należy udzielić pomocy rannemu. Wpływ na ciśnienie oraz tętno ma mnóstwo czynników, m.in. skala utraty krwi, morfina, adrenalina, zmęczenie, ból itp. 
+
+![Tabela ciśnień](../_data/guides/medical/pressure.png)
+
+Dla tętna przyjmuje się następujące wartości:
+- Niskie – poniżej 60
+- Normalne – 60 – 100
+- Wysokie – powyżej 100
+	
+	- Niskie ciśnienie, tętno - Przetoczyć krew, podać adrenalinę. Jeżeli doszło do przedawkowania morfiny, podać tą samą liczbę dawek adrenaliny i odczekać.
+	- Wysokie ciśnienie, tętno – Podać morfinę lub adenozynę
+	- Niskie ciśnienie, wysokie tętno – Przetoczyć krew. 
+	
+Należy zapamiętać najważniejszy próg ciśnienia oraz tętna – 50/60, tętno 40. Od tych wartości zaczyna istnieć jakakolwiek szansa na odzyskanie przytomności.
+
+## Leki
+
+- Adrenalina służy do podniesienia przede wszystkim tętna, a przy tym ciśnienia krwi. W większości przypadków będzie miała ona za zadania skontrować działanie morfiny, obniżające tętno w przypadku nieprzytomnego rannego.
+- Morfina jest skuteczym środkiem przeciwbólowym. Pozwala w zaledwie 30 sekund uśmierzyć ból, przez który ranny nie jest w stanie prowadzić celnego ostrzału. Skutki uboczne to obniżenie tętna oraz pośrednie obniżenie ciśnienia krwi (zmniejszona lepkość krwi).
+- APAP jest alternatywą dla morfiny. Nie powoduje praktycznie żadnych skutków ubocznych, aczkolwiek jest trochę słabszy w tłumieniu bólu oraz ma dłuższe opóźnienie w działaniu. Można podawać tylko przytomnym!
+
+| Cecha | Apap | Morfina |
+|-------|------|---------|
+| Maksymalny czas działania | 10 min | 30 min |
+| Opóźnienie działania | 2 min | 0.5 min |
+| Siła przeciwbólowa | 0.6 | 0.8 |
+| Zastosowanie | Ból u przytomnych osób | Duży ból lub u nieprzytomnych |
+| Efekty uboczne | Brak | Obniżone tętno |
+| Ryzyko nieprzytomności | Pomijalne | Wysokie, wystarczą 2 użycia |
```

Comment:
This and other tables in this file should be formatted to look like a table in file. Please look at `armaforces_medical.md`.